lib/IMAP/ImapMessageFetcher.php

```diff
@@ -139,6 +139,8 @@ public function fetchMessage(?Horde_Imap_Client_Data_Fetch $fetch = null): IMAPM
 			$this->hasAnyAttachment = $this->hasAttachments($structure);
 
 			$isEncrypted = $this->smimeService->isEncrypted($fetch);
+			$isOpaqueSigned = $structure->getType() === 'application/pkcs7-mime'
```

Comment:
Can we add a fallback for application/x-pkcs7-mime?

Reply:
Done